src/cli/constant.js

```diff
@@ -70,6 +70,16 @@ const categoryOrder = [
  * Note: The options below are sorted alphabetically.
  */
 const options = {
+  cache: {
+    type: "boolean",
+    default: false,
+    description: "Only format changed files",
+  },
+  "cache-location": {
+    type: "path",
+    description: "Path to the cache file or directory",
+    default: ".prettiercache",
```

Comment:
Default to `node_modules/.cache/prettier`? https://github.com/avajs/find-cache-dir, #5910 also use this.

Reply:
[9581f64](https://github.com/prettier/prettier/pull/12800/commits/9581f64d4c8dee80ca1411b51632bcfe6792a653)

Reply:
in my opinion, node_modules/.cache is a *really* bad place to put the cache.
Because when using yarn, if any dependency is upgraded (not just prettier), then .cache is cleared.
find-cache-dir does allow specifying a CACHE_DIR so I can use that to get around it, but if you use alot of packages and upgrade daily and use yarn, it make the cache option useless.